gen3-integration-tests/tests/test_gen3_workflows.py

```diff
@@ -26,10 +33,53 @@ class TestGen3Workflow(object):
     invalid_user = "dummy_one"
     s3_folder_name = "integration-tests"
     s3_file_name = "test-input.txt"
-    s3_storage_config = WorkflowStorageConfig.from_dict(
-        gen3_workflow.get_storage_info(user=valid_user, expected_status=200)
-    )
-    Gen3Workflow.empty_bucket_with_boto3(s3_storage_config)
+
+    @classmethod
+    def setup_class(cls):
+        # Ensure the bucket is wiped before running the tests
+        cls.gen3_workflow.delete_user_bucket()
+
+        cls.s3_storage_config = WorkflowStorageConfig.from_dict(
+            cls.gen3_workflow.get_storage_info(user=cls.valid_user, expected_status=200)
+        )
+
+    def _nextflow_parse_completed_line(self, log_line):
+        """
+        Parses a line from the nextflow log that indicates a completed task.
+        Extracts: task name, work directory (and protocol), exit code, and status.
+
+        :param str log_line: A line from the Nextflow log file.
```

Comment:
can you add an example input line here, and the associated example output

Reply:
Changes made. Could you take a look and resolve this comment if it looks good?